### PR TITLE
feat(ui): Always show "Suggested Owners" (APP-132)

### PR DESCRIPTION
**Pull request by billyvg**
* renamed "Suggested Assignees" --> "Suggested Owners"
* remove "Create Ownership Rule" from "Group Details Actions"
* allow "EmptyMessage" to pass "className" to styled component
* Add "Add Rule" button when there are no "Suggested Owners"
  * Disable "Add Rule" button when no "project:write" (and not visible w/o feature flag)
(The "Add Rule" button is not visible when there are committers or owners atm)

no `project:write` ~~(tooltips don't work on disabled buttons :()~~
image

image

Tweaked tooltip a bit:
image

### CODE REVIEW

**Comment by dcramer** [comment]
Quick questions:

1. Does the disabled button show a tooltip on why? I think I'm reading it does, and if so we should either 1. fix that or 2. hide the button
2. We're still planning to show this widget even when there are no existing owners, right?

**Comment by billyvg** [comment]
1) No tooltip because of mouse events on a disabled button
2) Yes, currently it is always visible (even without feature tag, which will only display committers w/o flag).

File: docs-ui/components/button.stories.js [hunk 1]
```diff
@@ -33,7 +33,19 @@ storiesOf('Buttons', module)
...
 
         <Item>
-          <Button to={''} disabled onClick={action('click disabled')}>
+          <Button to={'#href'} disabled onClick={action('click disabled')}>
+            Disabled Button
+          </Button>
+        </Item>
+
+        <Item>
+          <Button href="#href" disabled onClick={action('click disabled')}>
```
Comment (billyvg) [inline docs-ui/components/button.stories.js]: Added some different disabled Button types to storybook to check that nothing happens when you click a disabled Button

File: src/sentry/static/sentry/app/components/buttons/button.jsx [hunk 2]
```diff
@@ -51,15 +51,18 @@ class Button extends React.Component {
...
 
   // Intercept onClick and propagate
-  handleClick = (...args) => {
+  handleClick = (e, ...args) => {
     let {disabled, busy, onClick} = this.props;
 
     // Don't allow clicks when disabled or busy
-    if (disabled || busy) return;
+    if (disabled || busy) {
+      e.preventDefault();
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/buttons/button.jsx]: We don't *really* need to do this since we also strip the URLs

**Comment by benvinegar** [comment]
IMO, it shouldn't be "suggested" owners. It should just be "Owners" or "Issue Owners" (we should be consistent about this nomenclature).

You either own this code (as a result of configuration or because you authored the code recently) or you don't.

**Comment by benvinegar** [comment]
@billyvg Have you connected with @lynnagara on #8095 (which just landed)? Seems like we have multiple solutions to the same problem here.

**Pull request closed without merging**